test/get-config.test.js

```diff
@@ -114,19 +120,19 @@ test('Read options from package.json', async (t) => {
     tagFormat: `v\${version}`,
     plugins: false,
   };
+  const context = {cwd};
+  // Verify the plugins module is called with the plugin options from package.json
+  td.when(plugins({context, options}, {})).thenResolve(pluginsConfig);
   // Create package.json in repository root
   await outputJson(path.resolve(cwd, 'package.json'), {release: options});
 
-  const {options: result} = await t.context.getConfig({cwd});
+  const result = await t.context.getConfig(context);
 
-  const expected = {...options, branches: ['test_branch']};
   // Verify the options contains the plugin config from package.json
-  t.deepEqual(result, expected);
-  // Verify the plugins module is called with the plugin options from package.json
-  t.deepEqual(t.context.plugins.args[0][0], {options: expected, cwd});
+  t.deepEqual(result, {options, plugins: pluginsConfig});
 });
 
-test('Read options from .releaserc.yml', async (t) => {
+test.serial('Read options from .releaserc.yml', async (t) => {
```

Comment:
this file required `.serial` for some tests due to similar reasons as other test files because of conflicting stub setup with the plugins module